Problem setting a fix IP for IRMQTTServer

@crankyoldgit Hi,

I have a problem setting a fix IP for the module. See below error message during build.

In file included from .../IRMQTTServer.ino:311:0:
IRMQTTServer.h:62:7: error: 'IPAddress' does not name a type
              const IPAddress kIPAddress = IPAddress(192, 168, 1, 238);
IRMQTTServer.h:63:7: error: 'IPAddress' does not name a type
             const IPAddress kGateway = IPAddress(192, 168, 1, 254);
IRMQTTServer.h:64:7: error: 'IPAddress' does not name a type
            const IPAddress kSubnetMask = IPAddress(255, 255, 255, 0);
.../IRMQTTServer.ino: In function 'void setup_wifi()':
.../IRMQTTServer.ino:1829:36: error: 'kIPAddress' was not declared in this scope
           wifiManager.setSTAStaticIPConfig(kIPAddress, kGateway, kSubnetMask);
.../IRMQTTServer.ino:1829:48: error: 'kGateway' was not declared in this scope
          wifiManager.setSTAStaticIPConfig(kIPAddress, kGateway, kSubnetMask);
.../IRMQTTServer.ino:1829:58: error: 'kSubnetMask' was not declared in this scope
         wifiManager.setSTAStaticIPConfig(kIPAddress, kGateway, kSubnetMask);

Thanks!

_Originally posted by @ldellus in https://github.com/crankyoldgit/IRremoteESP8266/issues/834#issuecomment-520159663_

--- a/examples/IRMQTTServer/IRMQTTServer.h
+++ b/examples/IRMQTTServer/IRMQTTServer.h
@@ -5,6 +5,9 @@
 #ifndef EXAMPLES_IRMQTTSERVER_IRMQTTSERVER_H_
 #define EXAMPLES_IRMQTTSERVER_IRMQTTSERVER_H_
 
+#if defined(ESP8266)
+#include <ESP8266WiFi.h>
+#endif  // ESP8266
 #include <IRremoteESP8266.h>
 #include <IRrecv.h>
 #include <IRsend.h>

